gcp/rakefiles/vars.rb

```diff
@@ -36,13 +36,16 @@ def self.set_vars(env, project_type)
       zone = "#{ENV["USER"]}.#{env}.gcp.gpii.net."
       if ENV["TF_VAR_dns_zones"].nil?
         ENV["TF_VAR_dns_zones"] = %Q|{ #{env}-gcp-gpii-net = "#{zone}" }|
+        $compose_env << "TF_VAR_dns_zones"
       end
       if ENV["TF_VAR_dns_records"].nil?
         ENV["TF_VAR_dns_records"] = %Q|{ #{env}-gcp-gpii-net = "*.#{zone}" }|
+        $compose_env << "TF_VAR_dns_records"
       end
     end
 
-    ENV["ENV"] = env
+    ENV["ENV"] = ENV["TF_VAR_env"] = env
```

Comment:
Please add a test for this new logic.

Reply:
This affects only `docker-compose` config, we used to set it there:
https://github.com/gpii-ops/gpii-infra/blob/master/gcp/docker-compose.yaml#L46

Reply:
Not sure I follow. I meant that this function is covered by unit tests, and you changed the behavior of the function, so that change should be reflected by a change in the unit tests.

Not a huge deal. I leave it up to you.